src/plugins/intel_gna/src/transformations/pwl_approximation.cpp

```diff
@@ -150,7 +151,7 @@ double pivot_search(const details::Function<T>& activation_function,
             if (max_epsilon > max_epsilon_prev) {
                 j = j - 1;
                 Delta = Delta / 2;
-            } else if (max_epsilon == max_epsilon_prev) {
+            } else if (std::abs(max_epsilon - max_epsilon_prev) < EPSILON) {
```

Comment:
We have at least 3 `fp32eq` functions inside plugin. Can we put it in one place and use everywhere with the same epsilon value?

Reply:
Done, but some tests started to fail
https://openvino-ci.toolbox.iotg.sclab.intel.com/job/private-ci/job/ie/job/ie-tests-linux-ubuntu20-gna/30856/#showFailuresLink